ui/components/Header.js

```diff
@@ -94,7 +94,7 @@ class Header extends React.Component {
                 </Hidden>
                 <Grid item xs container alignItems="center" className={classes.pageTitleWrapper}>
                   <Typography color="inherit" variant="h5" className={classes.pageTitle}>
-                    {title}
+                    {title === 'Dashboard' ? 'System Dashboard' : title}
```

Comment:
Can this be renamed in the page route rather than a hack in the components?

Reply:
I have fixed it on page routes